src/pages/index.jsx

```diff
@@ -385,7 +385,7 @@ export async function getServerSideProps ({ query }) {
     if (
       stats
       && config.ALLOW_PLAYER_STATISTICS_FOR_BLACKLIST === false
-      && config.BLACKLISTED_CFTOOLS_IDS.includes(query.steam64)
+      && config.BLACKLISTED_STEAM64_IDS.includes(query.steam64)
```

Comment:
I'm assuming this is what made you raise the issue. But the truth is that, even though the query parameter is defined as `steam64`, we automatically resolve a bunch of id's behind the scenes by using the cftools-sdk `SteamId64` and `CFToolsId` constructors. We could possibly even support Bohemia Interactive and BattlEye ids - but I'm not sure if those are even used by normal players

I'll fix the naming convention - sorry about the confusion

Reply:
and bi id and be id is not really used in the community